src/wazuh_modules/wm_monitor.c

```diff
@@ -9,18 +9,20 @@
  * Foundation.
  */
 #include <stdlib.h>
-#include "../../wmodules_def.h"
-#include "wmodules.h"
 #include "wm_monitor.h"
+#include "wmodules.h"
 #include "defs.h"
 
+#define DEFAULT_NO_AGENT 0
+#define DEFAULT_DAY_WAIT -1
+
 static void* wm_monitor_main(wm_monitor_t *data);        // Module main function. It won't return
 static void wm_monitor_destroy(wm_monitor_t *data);      // Destroy data
-const char *WM_MONITOR_LOCATION = "monitor";            // Location field for event sending
+const char *WM_MONITOR_LOCATION = "monitor";             // Location field for event sending
 cJSON *wm_monitor_dump(const wm_monitor_t *data);
 
 const wm_context WM_MONITOR_CONTEXT = {
-    WM_MONITOR_LOCATION,
+    "monitor",
```

Comment:
Add define for line 21 and 25 text constant.

Reply:
Will do!